NDTensors/src/blocksparse/blocksparse.jl

```diff
@@ -104,7 +104,7 @@ end
 function copyto!(D1::BlockSparse, D2::BlockSparse)
   blockoffsets(D1) ≠ blockoffsets(D1) &&
     error("Cannot copy between BlockSparse storages with different offsets")
-  copyto!(data(D1), data(D2))
+  Base.copyto!(expose(data(D1)), expose(data(D2)))
```

Comment:
Is there a need for writing it as `Base.copyto!` instead of `copyto!`? I assume we are removing `NDTensors.copyto!`?

Reply:
Thanks for pointing this out. Its the same as before, I implemented the `Base.copyto!` while still having the `NDTensors.copyto!` then removed the NDTensors version to see if/where things failed to replace them with base. But I forgot to go back and remove the `Base.` Ill go through and get rid of them. Same as the `LinearAlgebra.` in the `mul!` call above